### ISSUE TITLE
metastore.delete force does not force

### ISSUE THREAD

**Issue by althrussell** [databricks/databricks-sdk-py#142]
I believe the issue is on the API side but 
`workspace_client.metastores.delete(id=metastoreId',force=True)` will not force delete the metastore 

I am having to send force as url parameter
`workspace_client.metastores.delete(id=metastoreId+'?force=true',force=True)` which is adding parameter force=true in url and not message body

**Comment by nfx** [comment]
@althrussell and adding it to URL works, right?..

**Comment by althrussell** [comment]
@nfx yes appending to the url works. I've had this issue direct with api

**Comment by nfx** [comment]
@althrussell I'm rounding off for this week, but I think the error is in the core.py, where `query` is not encoded properly. If you turn on debug logging, you'd probably see `?force=True`, but our backend doesn't understand `True`. Could you check if doing a small dictionary comprehension helps and make a PR?

```python
    def do(self, method: str, path: str, query: dict = None, body: dict = None, raw: bool = False) -> dict:
        headers = {'Accept': 'application/json', 'User-Agent': self._user_agent_base}
        response = self._session.request(method,
                                         f"{self._cfg.host}{path}",
                                         params=query,
                                         json=body,
                                         headers=headers)
```
